pageserver/src/tenant/mgr.rs

```diff
@@ -1375,7 +1375,7 @@ impl TenantManager {
         result
     }
 
-    #[instrument(skip_all, fields(tenant_id=%tenant_shard_id.tenant_id, shard_id=%tenant_shard_id.shard_slug(), new_shard_count=%new_shard_count.0))]
+    #[instrument(skip_all, fields(tenant_id=%tenant_shard_id.tenant_id, shard_id=%tenant_shard_id.shard_slug(), new_shard_count=%new_shard_count.literal()))]
```

Comment:
We wouldn't want an assert (because the shard count is external input that needs validating), but there is already a bail! a few lines down that rejects any count that is not an increase -- this implicitly also covers that case of new_shard_count=0 or new_shard_count=1 -- either one is an error and will be rejected.